### PR TITLE
CAPPS-5596 fix backups now that we are using pay per request

### PR DESCRIPTION
**Pull request by nicwise**
https://pushpay.atlassian.net/browse/CAPPS-5596

- [ ] Locally Pair Tested

### CODE REVIEW

File: lib/dynamo-backup.js [hunk 1]
```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
         if (err) {
             return callback(err);
         }
+        var limit = 1;
 
-        var readPercentage = self.readPercentage;
-        var limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
+        if (data.Table.BillingModeSummary.BillingMode == "PAY_PER_REQUEST") {
```
Comment (nicwise) [inline lib/dynamo-backup.js:163]: this is the main bit we care about

File: lib/dynamo-backup.js [hunk 2]
```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
         if (err) {
             return callback(err);
         }
+        var limit = 1;
```
Comment (qisaw) [inline lib/dynamo-backup.js:161]: `let`

File: lib/dynamo-backup.js (hunk 1 above)
Comment (qisaw) [reply lib/dynamo-backup.js:163]: Do we want PAY_PER_REQUEST in a constant?

File: lib/dynamo-backup.js [hunk 3]
```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
...
 
-        var readPercentage = self.readPercentage;
-        var limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
+        if (data.Table.BillingModeSummary.BillingMode == "PAY_PER_REQUEST") {
+            // default to 200 RCUs if we are in pay per request
+            limit = 200;
+        } else {
+            var readPercentage = self.readPercentage;
```
Comment (qisaw) [inline lib/dynamo-backup.js:167]: `const`

File: lib/dynamo-backup.js [hunk 4]
```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
...
+            limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
```
Comment (qisaw) [inline lib/dynamo-backup.js:168]: lol Math.max(false, 10) // 10
but Math.max(NaN, 10) // NaN 
🤦‍♂ JS sucks sometimes
Good pickup here :)

**Review by qisaw — APPROVED** [review]
LGTM, just some 💄

File: lib/dynamo-backup.js (hunk 4 above)
Comment (nicwise) [reply lib/dynamo-backup.js:168]: I didn't - thats the original code :)

**Pull request merged**